README.md

```diff
@@ -27,11 +27,94 @@ by the BigBlueButton HTML5 client to extend its functionalities.
 
 An overview of the plugin architecture and capabilities can be found [here](https://github.com/bigbluebutton/plugins/blob/main/README.md#capabilities-and-technical-details).
 
+### Building a Hello-World plugin example
+
+As an example, this secction will go through all the steps one need to follow in order to have a usable plugin, or at least a good starting point. In this case, the intended example plugin will add an option within the options dropdown menu (3 dots on top right-hand corner of your bbb-ui) that when clicked, it will trigger a pop up that says "hello world".
```

Comment:
```suggestion
As an example, this section will go through all the steps one need to follow in order to have a usable plugin, or at least a good starting point. In this case, the intended example plugin will add an option within the options dropdown menu (3 dots on top right-hand corner of your bbb-ui) that when clicked, it will trigger a pop up that says "hello world".
```